test.js

```diff
@@ -93,14 +93,3 @@ test('FastifyError.toString returns code', t => {
   const err = new NewError()
   t.is(err.toString(), 'FastifyError [CODE]: foo')
 })
-
-test('Create the error without the new keyword', t => {
```

Comment:
Why is this test removed?

Reply:
It moved to `class` implementation. It can't be able to use `Error()` for initialize.

Reply:
We have discussed in https://github.com/fastify/fastify-error/pull/61#issuecomment-1046734623. Well, I think it needs to be discussed within the fastify organization and make a decision.